### ISSUE TITLE
Cap flux coils causes crash

### ISSUE THREAD

**Issue by blitzmann** [DarkFenX/Pyfa#397]
```
Traceback (most recent call last):
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\gui\builtinViews\fittingView.py", line 65, in fitSelected
    view.fitSelected(event)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\gui\builtinViews\fittingView.py", line 287, in fitSelected
    self.slotsChanged()
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\gui\builtinViews\fittingView.py", line 439, in slotsChanged
    self.generateMods()
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\gui\builtinViews\fittingView.py", line 396, in generateMods
    fit = sFit.getFit(self.activeFitID)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\service\fit.py", line 254, in getFit
    self.recalc(fit, withBoosters=True)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\service\fit.py", line 996, in recalc
    fit.calculateModifiedAttributes(withBoosters=withBoosters)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\eos\saveddata\fit.py", line 510, in calculateModifiedAttributes
    item.calculateModifiedAttributes(self, runTime, False)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\eos\saveddata\module.py", line 597, in calculateModifiedAttributes
    effect.handler(fit, self, context)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\eos\effects\shieldcapacitymultiply.py", line 12, in handler
    fit.ship.multiplyItemAttr("shieldCapacity", module.getModifiedItemAttr("shieldCapacityMultiplier"))
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\eos\effectHandlerHelpers.py", line 247, in multiplyItemAttr
    self.itemModifiedAttributes.multiply(*args, **kwargs)
  File "C:\Users\Ryan\Copy\Git\blitzmann\Pyfa2\eos\modifiedAttributeDict.py", line 288, in multiply
    self.__multipliers[attributeName] *= multiplier
TypeError: unsupported operand type(s) for *=: 'float' and 'NoneType'
```

Dunno why. This isn't due to any recent work with CREST or Attribute Overrides from what I can tell. 

EDIT: actually, it seems the sisi merge into master caused it. Not sure yet what caused it before that

**Comment by blitzmann** [comment]
Problem commit: 5577e1976bbe3015bb6baf924f71fac54b79bdde

**Comment by blitzmann** [comment]
Apparently our eve database is missing some attributes for Flux Coils? The fuck?

**Comment by blitzmann** [comment]
So, the SDE itself seems to be missing this data, according to Steves conversions. I have a possible fix, but it won't fix the missing attributes.

**Comment by blitzmann** [comment]
Okay, think I got it sorted. 

I apparently missed this from the phobos output:

```
  [-] powerOutputMultiplier: 1.0
  [-] shieldCapacityMultiplier: 1.0
  [-] shieldRechargeRateMultiplier: 1.0
```

The cap flux coils are assigned effect that use these attributes. It seems the CCP had them set to `1` which effectively made them useless, so they removed them. The effect still looks for them anyway.

I made a fix that, if an attribute is not found, it simply uses `1` to avoid crash.

**Comment by blitzmann** [comment]
I changed it to do this instead:

```
        if multiplier is None:
            return
```

I don't know what I didn't do this before, but this prevents the multiplier from doing anything if it doesn't even exist (which fixes #458 as well)
